js/layout-javascript/news-feed-code.js

```diff
@@ -1435,7 +1435,7 @@ DynamicList.prototype.renderLoopHTML = function () {
           var selector = $lineClamp.length ? '[data-line-clamp]' : '.news-feed-item-description';
           var linesToClamp = $lineClamp.length ? $lineClamp.data('line-clamp') : 3;
 
-          window.ellipsed.ellipsis(selector, linesToClamp);
+          window.ellipsed.ellipsis(selector, linesToClamp, {delimiter: '',});
```

Comment:
@upplabs-alex-levchenko @MaksymShokin 

1. Objects defined in a single line should always have spaces inside the `{}` brackets.
1. No trailing comma in objects or arrays

```suggestion
          window.ellipsed.ellipsis(selector, linesToClamp, { delimiter: '' });
```